ChangeHistory

```diff
@@ -7,7 +7,8 @@ Features and bug fixes are tagged on the 'develop' branch using the JIRA
 ticket identifier, "G4CMP-nnn", or using the GitHub pull request, "PR-nn".
 
 Development Project for G4CMP-515
-2025-10-15  G4CMP-515 : Expand charge reflection for both specular and diffuse.
+2025-11-20  G4CMP-515 : Expand charge reflection for both specular and diffuse.
+2025-11-20  G4CMP-511 : Create parallel Lambertian reflection code for charges.
```

Comment:
Your modifications to G4CMPBoundaryUtils etc. have History entries dated "20251204" (although you committed them yesterday).  This top level date should always reflect the most recent chages.  Since you committed a bunch of stuff yesterday, these dates should both have been set to "2025-12-09".  

I know it is really bureaucratic, but if we have to go back a year from now, having the correct information will make it much easier to find the relevant commits and understand how they were made.

Reply:
Oh, sorry I just misunderstood your original comment. My bad!